app/enterprise/1.3-x/kong-cloud.md

```diff
@@ -1,12 +1,74 @@
 
-Kong Cloud enforces HTTPS for all of the services that it operates, including Admin API, Kong Manager, and Kong Developer Portal. Kong Cloud does not enforce HTTPS for traffic destined for the customer's proxy, since certificate management and associated domains are under the customer's control. To enforce HTTPS on upstream traffic, use the [certificate](https://docs.konghq.com/enterprise/{{page.kong_version}}/admin-api/#certificate-object) and [SNI](https://docs.konghq.com/enterprise/{{page.kong_version}}/admin-api/#sni-object.) objects through the Admin API.
+# Kong Cloud  differences from on-premise
+Kong Cloud, while being our hosted Kong Enterprise solution, has a few key differences from a self-hosted, on-premise Kong Enterprise cluster.
 
-For non-proxy traffic, Kong Cloud is the terminus for the request, and Kong controls the protocol and shape of traffic carefully. Additionally, Kong generally considers non-proxy traffic to be sensitive (e.g., Admin API requests, login credentials to Kong Manager). Kong Cloud accomplishes enforcement through HTTP → HTTPS redirects, and the use of the HSTS response header. Because Kong Cloud controls the domain hosting the endpoints for these services, e.g., `https://manager-client.kong-cloud.com`, Kong Cloud maintains the TLS certificate for this service since Kong owns the domain.
+## Security
+By default, RBAC is enabled on both the Kong Manager and Kong Gateway Admin API to prevent unauthorized access to your Kong Enterprise cluster.
 
-For dedicated production Kong Cloud clusters (e.g., clusters setup for paying customers), proxy traffic is funneled through a network load balancer to Kong nodes, which then pass the request to the customer's upstream. Kong Cloud does not enforce any protocol or application layer behaviors here, because this traffic would be specific to the customer's upstream APIs. Thus, securing proxy traffic on a production Kong Cloud cluster is the customer's responsibility (e.g., setting up TLS certificates through Kong's Admin API/Manager). Customers generally use their own domain name for production API (e.g., api.example.com), so Kong does not provide a TLS certificate to use out-of-the-box for production proxy traffic. Kong would have no way to enforce TLS traffic and no manageable certificate/key to provide. Thus, securing production proxy traffic via TLS is the customer's responsibility.
+A root account named **kong_admin** will be created and provisioned on sign up.
+
+## Kong Manager
+In Kong Manager, some parts of the interface that aren’t relevant for a hosted setting due to being either irrelevant, ephemeral or inaccessible from outside of the private network are hidden or disabled. These include license details, database details, and cluster node details.
```

Comment:
simplify sentence structure, disambiguate "these," consolidate redundancy.

```suggestion
In Kong Manager, some parts of the interface are hidden or disabled since they aren’t relevant for a hosted setting due to being irrelevant, ephemeral, or inaccessible from outside of the private network. These hidden parts include details about the license, database, and cluster node.
```